vue/components/ui/molecules/button-icon-dropdown/button-icon-dropdown.stories.js

```diff
@@ -30,11 +30,12 @@ storiesOf("Components/Molecules/Button Icon Dropdown", module)
                         target: "_blank"
                     }
                 ]
-            }
+            },
+            disabled: false

```

Comment:
this could be controllable in the story, like `disabled: boolean("Disabled", false)`